### ISSUE TITLE
Camera permission is not requested

### ISSUE THREAD

**Issue by IzzySoft** [digitalfabrik/ehrenamtskarte#430]
**Describe the bug**
The `CAMERA` permission is not requested, and thus cannot be granted either. This makes it impossible to scan the Ehrenamtskarte, for example.

**To Reproduce**
1. Go to 'Ausweisen'
2. Click on 'Jetzt Aktivierungscode einscannen'
3. See the camera view open
4. Window stays black, no camera.

**Expected behavior**
Camera view is shown so the card can be scanned (question remains what the activation code is; I do not remember having received any with my card).

**Screenshots**
From a black screen? :wink:

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] n/a
 - Browser [e.g. chrome, safari] n/a
 - Version [e.g. 22] n/a

**Smartphone (please complete the following information):**
 - Device: Fairphone 2
 - OS: Android 10 (LineageOS)
 - Browser n/a
 - Version 2.0.2

**Additional context**
None so far.

**Comment by maxammann** [comment]
Thanks for opening this bug. Right now this is intentional, because we do not yet have build flavors.
The currently released version in the Play Store should not have this permission.

We can use this issue to re-add the camera permission as soon as we have build flavors.

**Comment by IzzySoft** [comment]
I see. But if that's intentional, shouldn't that tab be hidden entirely? It's a bit confusing if I'm asked to scan a code and there's no camera to do it :wink:

**Comment by maxammann** [comment]
Closing because it will be fixed by  #524.

**Comment by michael-markl** [comment]
Afaik, this is not done yet although the linked issue is closed. Let's put this as a technical task.

**Comment by maxammann** [comment]
Yes we basically need to enable the camera permission based on the build config

**Comment by maxammann** [comment]
Closing as soon all flavors of this app will have the camera perm. enabled.
